simpleast-core/src/main/java/com/discord/simpleast/code/CodeRules.kt

```diff
@@ -41,7 +41,7 @@ object CodeRules {
       Pattern.compile("""^```(?:([\w+\-.]+?)?(\s*\n))?([^\n].*?)\n*```""", Pattern.DOTALL)
 
   val PATTERN_CODE_INLINE: Pattern =
-      Pattern.compile("""^`(?:\s*)([^\n].*?)\n*`""", Pattern.DOTALL)
+      Pattern.compile("""^`([^`]*?)`""", Pattern.DOTALL)
```

Comment:
updates the regex to grab anything other than a tick. This mimics what we do on desktop where whitespaces are captured as well.